.circleci/config.yml

```diff
@@ -190,6 +194,40 @@ commands:
           paths:
             - android/app/build/outputs
 
+  android-e2e-test:
```

Comment:
We don't need this code on this PR.
All CircleCI related code should be on #3344, right?
So can you remove it?

Reply:
Yep, done